Test/ArpTest.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using NUnit.Framework;
+using SharpPcap;
+using SharpPcap.LibPcap;
+
+namespace Test
+{
+    [TestFixture]
+    public class ArpTest
+    {
+        [Test]
+        public void TestArp([PcapDevices] DeviceFixture fixture)
+        {
+            var d = (LibPcapLiveDevice)fixture.GetDevice();
```

Comment:
use `TestHelper.GetPcapDevice()` it will return an Ethernet pcap device where ARP should work without a try/catch

Reply:
is there any value in testing across each device? I suppose not?

Reply:
You are not likely to find more than 1 Ethernet adapter connected to the same server/pc